windows/src/global/delphi/general/CompilePackage.pas

```diff
@@ -170,14 +170,26 @@ function TCompilePackage.BuildKMP: Boolean;
 
     kmpinf.Assign(pack);
     kmpinf.RemoveFilePaths;
-    kmpinf.FileName := FTempPath + '\kmp.inf';
 
     psf := TPackageContentFile.Create(kmpinf);
     psf.FileName := 'kmp.inf';
     psf.Description := 'Package information';
```

Comment:
How about this description?
`psf.Description := 'Package information (INF)';`

Reply:
I didn't want to change the existing description, because there may be others using it.